### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by Xaeroxe**

### CODE REVIEW

File: examples/01_window/resources/config.ron [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-dimensions: null
-fullscreen: false 
-max_dimensions: null
-min_dimensions: null
-multisampling: 1
-title: "Windowing example"
-visibility: true
-vsync: true
+
+DisplayConfig (
```
Comment (torkleyy) [inline examples/01_window/resources/config.ron:2]: The recommended RON formatting doesn't have any spaces between `DisplayConfig` and `(`. Also, I think it's clear this is the display config, so we could skip the type.

**Review by torkleyy — COMMENTED** [review]
Nice, found one minor nit.

**Comment by torkleyy** [comment]
Oh well, it's fine, so I'll merge as is.

**Comment by Xaeroxe** [comment]
haha, you merged right as I finished fixing it.  Do you still want this commit?

**Comment by torkleyy** [comment]
Oops, sorry. No, let's just leave it as is.

**Pull request merged**